sherpa/astro/io/__init__.py

```diff
@@ -17,16 +17,34 @@
 #  with this program; if not, write to the Free Software Foundation, Inc.,
 #  51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
 #
-"""
-Provide Astronomy-specific I/O routines for Sherpa.
+"""Provide Astronomy-specific I/O routines for Sherpa.
 
 This module contains read and write routines for handling FITS [1]_
 and ASCII format data. The actual support is provided by
 the selected I/O backend package (currently Crates, provided
 by CIAO, or the FITS support in the AstroPy package, if installed).
 
-This module should not be imported directly if there is no
-I/O backend available.
+Which backend is used?
+----------------------
+
+When this module is first imported, Sherpa tries to import the
+backends installed with Sherpa in the order listed in the
+``sherpa.rc`` or ``sherpa-standalone.rc`` file. The first module that imports
```

Comment:
I wonder if it's better to just say `Sherpa configuration file` rather than give the full names.

If you do give the names then we want

```suggestion
``.sherpa.rc`` or ``.sherpa-standalone.rc`` file. The first module that imports
```

Reply:
Although I guess the file name depends on whether we are reading in the user's version (which has the leading `.`) or the system version (which is dotless). Which makes me think just referring to the configuration file (and possibly mentioning `sherpa.get_config()`) is the better solution.

Reply:
I'm worried that users don't know what the "configuration file" is, in particular since we have two. I thought about "``sherpa.rc`` (if you installed sherpa through CIAO) or ``sherpa-standalone.rc`` (if you installed sherpa as a Python package independent of CIAO)" but found that too verbose. At the very least, when giving both names, users know what name to search for if they don't know the location of this file and they know there is more than one, so if one does not work they will try the other.